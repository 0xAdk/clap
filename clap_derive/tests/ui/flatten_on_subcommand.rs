#[derive(clap::Clap)]
struct Opt {
    #[clap(flatten)]
    sub: SubCmd,
}

#[derive(clap::Clap)]
enum SubCmd {}

fn main() {}
