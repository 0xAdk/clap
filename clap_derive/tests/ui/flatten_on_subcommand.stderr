error[E0277]: the trait bound `SubCmd: clap::Args` is not satisfied
 --> $DIR/flatten_on_subcommand.rs:3:12
  |
3 |     #[clap(flatten)]
  |            ^^^^^^^ the trait `clap::Args` is not implemented for `SubCmd`
  |
  = note: required by `augment_args`
